src/cloud-functions.ts

```diff
@@ -493,5 +495,12 @@ export function optionsToTrigger(options: DeploymentOptions) {
   if (options.maxInstances) {
     trigger.maxInstances = options.maxInstances;
   }
+
+  if (options.vpcConnector) {
+    trigger.vpcConnector = options.vpcConnector;
+    trigger.vpcConnectorEgressSettings =
+      options.vpcConnectorEgressSettings || 'ALL_TRAFFIC';
```

Comment:
```suggestion
    if (options.vpcConnectorEgressSettings) {
      trigger.vpcConnectorEgressSettings = options.vpcConnectorEgressSettings;
    }
```

The Cloud Functions default for this is `PRIVATE_RANGES_ONLY` and I don't want to change those defaults without a good reason. If we only conditionally set the value it will use the backend default which is what I'd expect.

Reply:
Yeah I wasn't quite sure about setting up a default either. If indeed GCP does it by default it's a better choice I guess.
I made the changes.